packages/react-core/src/spatial-filter/helpers.js

```diff
@@ -0,0 +1,159 @@
+import turfIntersects from '@turf/boolean-intersects';
+
+export function applySpatialFilterToPoints(
```

Comment:
I think that in this file we're repeating a lot of code that we have in viewportFeaturesBinary you can get the point, line and polygon using this https://github.com/CartoDB/carto-react/blob/master/packages/react-core/src/filters/viewportFeaturesBinary.js#L29
and this
https://github.com/CartoDB/carto-react/blob/master/packages/react-core/src/filters/viewportFeaturesBinary.js#L30